MonoGame.Framework/Desktop/OpenTKGamePlatform.cs

```diff
@@ -88,13 +88,13 @@ class OpenTKGamePlatform : GamePlatform
 		private OpenALSoundController soundControllerInstance = null;
         // stored the current screen state, so we can check if it has changed.
         private bool isCurrentlyFullScreen = false;
-        private Toolkit toolkit;
         private int isExiting; // int, so we can use Interlocked.Increment
+
+        int windowDelay = 11;
```

Comment:
Is there really not a better way to know if the `OpenTKGameWindow` is safe to destroy than waiting for exactly 11 `ProcessEvents` calls?